dist/Rotation.js

```diff
@@ -10,117 +10,3 @@ $('#btn3').toggle(function() {
   $(this).rotate({ endDeg:-45 });
```

Comment:
javascript files should be written in camelCase https://www.w3schools.com/js/js_conventions.asp